.gitattributes

```diff
@@ -0,0 +1 @@
+core/pkg/fixhandler/testdata/inserts/*-crlf-newlines.yaml text eol=crlf
```

Comment:
[Question]: Does this mean we support only CRLF line endings? If not, why is it necessary to explicit this attribute?